<launch>
  <node name="circle_detection" pkg="task_buoy" type="circle_detection" respawn="true"  output="screen">
    <param name="t1minParam" type="int" value="178" />
    <param name="t1maxParam" type="int" value="183" />
    <param name="t2minParam" type="int" value="234" />
    <param name="t2maxParam" type="int" value="260" />
    <param name="t3minParam" type="int" value="227" />
    <param name="t3maxParam" type="int" value="255" />
  </node>
</launch>
